src/Component/Banner/banner.scss

```diff
@@ -0,0 +1,58 @@
+.banner_main {
+    padding: 40px 0 90px 0;
+    position: relative;
+    &::after {
+        content: "";
+         position: absolute;
+        width: 297px;
+        height: 370px;
+         right: 0;
+         top: -77px;
+         z-index: -1;
+         background: url("../../Assets/Img/midil.png");
```

Comment:
Naming convention, Assets v/s `assets` , Img v/s `images`.
Also try to name files appropriately